packages/repco-frontend/app/lib/usePlayQueue.tsx

```diff
@@ -0,0 +1,67 @@
+import { useContext, useEffect, useState } from 'react'
+import { ContextManager, Entity } from '~/lib/LocalStorageContext'
+import { Playlist, Track } from '../components/player/usePlaylists'
+
+export function useQueue() {
```

Comment:
For files that import a single component, or a single main component plus related smaller components, I'd propose that the file name is the same as the component name (in whatever casing scheme we decide) - so if we use kebab-case (which I proposed above), this would be either be `import { useQueue } from '~/lib/use-queue` or `import { usePlayQueue } from '~/lib/use-play-queue`. Not super important but I think in the long run this consistency helps.